src/helpers/index.ts

```diff
@@ -140,3 +140,17 @@ export async function installPackage(dependency: DependencyResolved): Promise<vo
   error.stack = stderr.trim()
   throw error
 }
+
+
+export async function getResolvedDependency(item: string | Dependency): Promise<DependencyResolved> {
+  // string entry
+  if (typeof item === 'string') {
+    return { name: item, directory: await resolveDependencyPath(item) }
+  }
+
+  if ('name' in item) {
+    return { ...item, directory: await resolveDependencyPath(item.name) }
+  }
+
+  throw Error("The package-deps entry is not valid")
```

Comment:
```suggestion
  throw Error(`The package-deps entry ${item} is not valid. Please see https://github.com/steelbrain/package-deps#how-it-works`)
```

Reply:
I am down for this, but please also log the exact invalid entry to the console

Reply:
I am confused. When we throw an error, its message gets logged automatically. There is no need for a separate `console.error`. Am I understanding this right?

Reply:
Yes, you are understanding it correctly, but if it makes sense, imagine if that `item` was somehow a dump of package.json

I vote that we remove it from the error message, and log it to console

Reply:
Oh, OK. That's for making the error message more succinct.